packages/contact-center/store/src/store.ts

```diff
@@ -101,7 +101,7 @@ class Store implements IStore {
         this.agentId = response.agentId;
         this.wrapupCodes = response.wrapupCodes;
         this.isAgentLoggedIn = response.isAgentLoggedIn;
-        this.deviceType = response.deviceType ?? 'AGENT_DN';
+        this.deviceType = response.deviceType ?? this.loginOptions[0];
```

Comment:
The reason behind hardcoding these values and defaulting to AGENT_DN was that each time, it came from the backend, they were in random orders. 

Did you check if all 3 options are enabled, AGENT_DN was the zeroth all the time?

Reply:
I had noticed that the list is randomized, but it will always be the 0th element of the Select as well. Should I sort loginOptions itself? (alphabetically or a hardcoded custom order)